type/__apt_source/files/source.list.template

```diff
@@ -2,13 +2,14 @@
 set -u
 
 entry="$uri $distribution $component"
+
 cat << DONE
 # Created by cdist ${__type##*/}
 # Do not change. Changes will be overwritten.
 #
 
 # $name
-deb ${forcedarch} $entry
+deb ${options} $entry
 DONE
 if [ -f "$__object/parameter/include-src" ]; then
    echo "deb-src $entry"
```

Comment:
I think that `signed-by` should also be set for `deb-src` lines.

cf. e.g. https://support.torproject.org/apt/tor-deb-repo/